repository/src/main/java/com/astasko/pdp/repository/impl/UserRepositoryImpl.java

```diff
@@ -0,0 +1,61 @@
+package com.astasko.pdp.repository.impl;
+
+import com.astasko.pdp.domain.model.User;
+import com.astasko.pdp.repository.api.UserRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.simple.SimpleJdbcInsert;
+
+import java.util.HashMap;
+import java.util.Random;
+
```

Comment:
add @Repository annotation instead of context creation. It's a SpringBoot. There are no need to configure context